source/includes/trip-api.md

```diff
@@ -1203,6 +1204,97 @@ Status code | Description
 404         | Booking not found.
 409         | Trying to update a cancelled booking.
 
+## Tracking endpoint
+
+This endpoint allows you to retrieve the latest position of driver(s) assigned to the trip. Not every trip is guaranteed to have driver tracking.
+
+> To get driver position(s) of a trip currently in progress, use the following call:
+
+```bash
+curl https://api.staging.mydaytrip.net/partners/v3/trip/tracking/bookingId 
+  -H "x-api-key: your-api-key" 
+```
+
+```javascript
+
+```
+
+```python
+
+```
+
+> Make sure to replace `bookingId` with the real booking id.
+
+> Example response for a trip served by one driver:
+
+```json
+{
+   "driverPositions": [
+      {
+         "timestamp": "2023-12-04T18:00:00Z",
+         "driverId": "4381b73d-be1d-4d7d-bf7c-cdde5292c1b9",
+         "position": {
+            "lat": 50.09298565485293,
+            "lon": 14.453347220812443
```

Comment:
I would suggest to go with standard naming, either `latitude` `longitude` or `lat` `lng`

Reply:
We are already using `lat` and `lon` in other endpoints so I have to keep it consistent.

Reply:
I see. It's not a big deal, so let's stick to a consistent naming.